client/src/components/pages/common/Home/index.js

```diff
@@ -1,5 +1,17 @@
 import React from 'react';
+import DotsStepper from '../../../application/DotsStepper';
+import registrationSteps from '../../../application/DotsStepper/registrationSteps.json';
+import draw1 from '../../../../assets/images/draw1.svg';
```

Comment:
change image name to be `HomeImg` instead if draw1
```suggestion
import HomeImg from '../../../../assets/images/HomeImg.svg';
```